pkg/crypto.go

```diff
@@ -70,5 +80,32 @@ func (c Crypto) Setup(hmacFile string, masterFile string) (bool, error) {
 }
 
 func (c Crypto) Protect(value string) (string, error) {
-	return value, nil // TODO implement this
+	var form = map[string]any{"datum": value}
+	log.Infof("%s > Protecting text using crypto.", c.instance.ID())
+	requestFormData := c.instance.http.RequestFormData(form)
+	response, err := requestFormData.Post(EncryptJsonPath)
+
+	if err != nil {
+		return "", fmt.Errorf("%s > Error while encrypting text using crypto: %s", c.instance.ID(), err)
+	} else if response.IsError() {
+		return "", fmt.Errorf("%s > cannot encrypt text using crypto: %s", c.instance.ID(), response.Status())
+
+	}
+
+	bodyReader := response.RawBody()
+
+	defer func(bodyReader io.ReadCloser) {
+		err := bodyReader.Close()
+		if err != nil {
+			log.Errorf("%s > cannot close crypto response body reader: %s", c.instance.ID(), err)
+		}
+	}(bodyReader)
+
+	var body CryptoResponseBody
+
+	if err = fmtx.UnmarshalJSON(bodyReader, &body); err != nil {
+		return "", fmt.Errorf("%s > cannot unmarshal crypto response body: %s", c.instance.ID(), err)
```

Comment:
Unmarshal is too techy word again. "Parse response" I am using in other places. Please align the style